src/node.js

```diff
@@ -73,6 +76,30 @@
         NativeModule.require('_third_party_main');
       });
 
+    } else if (process.openManPage) {
+      // Attempt to spawn `man 1 node`.
+      const cp = NativeModule.require('child_process');
```

Comment:
Could this whole case be a function?
(Just a nit, not blocking a merge, the whole function is pretty messy as is)